pronotepy/pronoteAPI.py

```diff
@@ -259,6 +265,9 @@ def _parse_html(html: bytes) -> dict:
         for attr in onload_c.split(","):  # type: ignore
             key, value = attr.split(":")
             attributes[key] = value.replace("'", "")
+
+        if "MR" not in attributes or "ER" not in attributes:
+            return _Communication._parse_html(html)
```

Comment:
Wouldn't this cause a `MaxRecursionError` in case the if statement is true?

Reply:
Maybe we should just raise a `PronoteAPIError`

Reply:
Yes It will but It's better than a While loop because this prevents it from blocking the program

Reply:
No we can't because it's a random error from Pronote so it's bad for the client to get an error due of an error in pronote

Reply:
Yeah but that still sends an error, even a more confusing one... I'd rather catch a pronotepy exception than a random `MaxRecursionError` exception. We really can't do anything about it, so I guess we should just tell the client that something bad happened.

Reply:
Okay I will add a try except with MaxRecursionError

Reply:
I mean why would we call the function again if it failed the first time? None of the inputs changed, so the output will be the same -> will loop infinitely.

Reply:
yes indeed I was wrong I basically wanted to redo a request to get a new html